pkg/distributor/distributor.go

```diff
@@ -45,6 +45,21 @@ var (
 		Name:      "distributor_received_samples_total",
 		Help:      "The total number of received samples.",
```

Comment:
Update this description to mention doesn't include rejected samples or deduped samples.